docs/connection-and-channel-management.md

```diff
@@ -0,0 +1,53 @@
+# Connection and channel management
+
+Spaces SDK uses the Ably SDK client [connections](https://ably.com/docs/connect?lang=javascript) and [channels](https://ably.com/docs/channels?lang=javascript) to provide higher level features like cursors or locations. Both connections and channels will transition through multiple states throughout their lifecycle. Most state transitions (like a short loss in connectivity) will be handled by the Ably SDK, but there will be use cases where developers will need to observe these states and handle them accordingly.
```

Comment:
Interesting that we're already seeing naming issues here. I think we should refer to the Ably SDK as the Ably Core SDK. I need to speak with @m-hulbert about this more broadly though.

Reply:
I'll update this to Ably Core SDK unless there's been any new developments here